### ISSUE TITLE
second line of text below a characteristic

### ISSUE THREAD

**Issue by thncode** [simont77/fakegato-history#104]
What drives an accessory within the EVE app to show a second line of text below the name - like power below the outlet name - and how can that be changed to another value?

**Comment by NorthernMan54** [comment]
Can you share a screenshot to explain this better

**Comment by bwp91** [comment]
![79671292-7F6E-4C71-9365-310E10AA175D](https://user-images.githubusercontent.com/43026681/98464554-a83b6d80-21bb-11eb-84e2-eebd224fa25b.jpeg)

Maybe this? I’m not sure if it can be changed

**Comment by thncode** [comment]
Yes, I mean exactly that - but how is it triggered?

**Comment by NorthernMan54** [comment]
To be honest, no clue.  I looked at my device and I don't have that line of text available.  My only guess is that it is the consumption characteristic, ie E863F10D

**Comment by ebaauw** [comment]
Yes, it's _Consumption_.  It's shown when this characteristic is exposed on an _Outlet_ service.

I have not been able to detect any logic as to under what conditions Eve shows which characteristics as second line.  It shows humidity and air pressure under _Temperature_ for a (fake) Eve Weather sensor (with three separate services), and it even shows some of my custom characteristics for some of my custom services.  Sometimes editing the order of characteristics in the services screen seems to influence this (and sometimes Eve just crashes when trying that).

![IMG_0228](https://user-images.githubusercontent.com/22179355/98470080-abe2ea80-21e3-11eb-847d-68ddd325b902.png)

**Comment by thncode** [comment]
I can confirm that - I just thought it is just me not to understand the logic behind...

**Comment by 8191** [comment]
Would it make sense to expose the shelly plugs (SHPLG-1, SHPLG2-1, SHPLG-S, SHPLG-U1, i.e. the `ShellyPlugFactory`) per default as an outlet to allow showing up that second line?
